ROADMAP.md

```diff
@@ -1,45 +1,40 @@
-# Roadmap
+# Apollo iOS Roadmap - _Last Updated July 2021_
 
-This document is meant to give the community some idea of where we're going with the iOS SDK in the short and longer term. 
+Releases adhere to the [Semantic Versioning Specification](https://semver.org/). Under this scheme, version numbers and the way they change convey meaning about the underlying code and what has been modified from one version to the next.
 
-This document was last updated on January 4th, 2020 
+## 0.x - _Current_
 
-^ If that's more than three months ago, please file an issue asking [@designatednerd](https://github.com/designatednerd) to update this document. 😃
```

Comment:
Personally, I'd leave in the thing about annoy us with an issue if it's been more than 3 months since it was last updated, just to keep us honest, but that's up to you and @AnthonyMDev

Reply:
For sure, never a bad thing to have accountability. Done in 4b6d88d0.